src/pages-and-resources/discussions/data/slice.js

```diff
@@ -4,9 +4,11 @@ import { createSlice } from '@reduxjs/toolkit';
 export const LOADING = 'LOADING';
 export const LOADED = 'LOADED';
 export const FAILED = 'FAILED';
+export const SAVING = 'SAVING';
+export const SAVED = 'SAVED';
 
 const slice = createSlice({
-  name: 'appList',
+  name: 'discussions',
```

Comment:
The appList slice had most of what we needed, so it was moved up and augmented with saveStatus so that it held everything necessary for both AppList and AppConfigForm.